crates/primitives/chain-id/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "mp-chain-id"
+version.workspace = true
+edition.workspace = true
+license = "MIT"
+description = "Primitives for Madara digest logs"
+authors = { workspace = true }
+repository = { workspace = true }
+
+[package.metadata.docs.rs]
+targets = ["x86_64-unknown-linux-gnu"]
+
+[dependencies]
+
+# Madara Primitives
+mp-felt = { workspace = true }
+mp-hashers = { workspace = true }
+mp-transactions = { workspace = true }
+# Starknet dependencies
+blockifier = { workspace = true }
+starknet-ff = { workspace = true }
+starknet_api = { workspace = true }
```

Comment:
I think you don't need all of those dependencies.
Only `starknet-ff` and `mp-felt`